src/App.js

```diff
@@ -1,18 +1,22 @@
 import './App.css';
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import Bookshelf from './components/bookShelf';
 import Nav from './components/nav';
 import Categories from './components/categories';
+import store from './redux/store';
 
 const App = () => (
-  <BrowserRouter>
-    <Nav />
-    <Routes>
-      <Route path="/" element={<Bookshelf />} />
-      <Route path="/categories" element={<Categories />} />
-    </Routes>
-  </BrowserRouter>
+  <Provider store={store}>
+    <BrowserRouter>
+      <Nav />
+      <Routes>
+        <Route path="/" element={<Bookshelf />} />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
+    </BrowserRouter>
+  </Provider>
```

Comment:
- [ ] Almost there! However, it appears that the `Provider` component has been wrapped inside the `App` component, whereas the project requirements specify that the `Provider` component should wrap the `App` component. Wrapping the App component inside the Provider component is an important step in ensuring that the application state is properly managed and made available to all components. Please adjust your code to wrap the App component with the Provider component instead. Thank you for your attention to this matter.
- This is how it should look 👇
```jsx
ReactDOM.render(
  <Provider store={store}>
    <App />
  </Provider>,
  document.getElementById("root")
)

```